json-schema/schema.json

```diff
@@ -34,6 +34,37 @@
               }
             }
           }
+        },
+        {
+          "if": {
```

Comment:
Is this schema saying if stac extensions includes both renders and web-map-links, a render sub-property is required? Why do you think it should be required? Seems like an optional property.

Reply:
Also, I may be mis-reading the schema but does the then statement mean that there must be a link object in the links list that has "rel": "render" - in which case does the example above needs updating to include a link with "rel": "render"?